pallets/collator-assignment/src/lib.rs

```diff
@@ -401,18 +411,24 @@ pub mod pallet {
                     )
                 };
 
+            log::trace!(
```

Comment:
I guess it would be ebtter if we haved a target for this logs, otherwise we always have to enable all debug no??